src/libs/Violations/ViolationsUtils.ts

```diff
@@ -162,6 +164,17 @@ function getTagViolationsForMultiLevelTags(
     return getTagViolationForIndependentTags(policyTagList, filteredTransactionViolations, updatedTransaction);
 }
 
+/**
+ * Returns a dot-separated string of violation messages for missing tag levels in a multi-level tag, based on error indexes.
```

Comment:
NAB, but I know we've used the term "dot separator" for things like `Jul 17 • Cash • Pending match` 
<img width="317" height="317" alt="image" src="https://github.com/user-attachments/assets/cbcf4b88-d6e9-4b63-90e8-aaba06456c80" />

```suggestion
 * Returns a period-separated string of violation messages for missing tag levels in a multi-level tag, based on error indexes.
```